chrome/content/otr.js

```diff
@@ -276,6 +284,18 @@ var otr = {
     return human.readString();
   },
 
+  base64encode: function(data, dataLen) {
+    let buf = ctypes.char.array(Math.floor(((dataLen+2)/3)*4 + 1))();
+    let size = libOTR.otrl_base64_encode(buf, data, dataLen);
+    return buf.readString();
+  },
+
+  base64decode: function(base64data) {
+    let buf = ctypes.unsigned_char.array(Math.floor((base64data.length+3)/4)*3)();
```

Comment:
Ok, yeah, you're right, but in both these the remainder is dropped,

https://github.com/off-the-record/libotr/blob/c9710e9b5bbd2f9fd540c6f4fe040937444d20e2/src/b64.h#L33-L35
https://github.com/off-the-record/libotr/blob/c9710e9b5bbd2f9fd540c6f4fe040937444d20e2/src/b64.c#L202

so the one above shouldn't have the `+ 1` then.